Sources/Card/Models/ThreeDSecureRequest.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+
+public struct ThreeDSecureRequest {
+
+    /// Specificy to always launch 3DS or only when required
+    public let sca: SCA
+
+    /// Url to return after flow has completed
+    public let returnUrl: String
+
+    /// Url to return when flow is cancelled
+    public let cancelUrl: String
+
+    /// Creates an instance of a card request
+    /// - Parameters:
+    ///   - sca: Specificy to always launch 3DS or only when required
+    ///   - returnUrl: Url to return after flow has completed
+    ///   - cancelUrl: Url to return when flow is cancelled
+    public init(sca: SCA, returnUrl: String, cancelUrl: String) {
+        self.sca = sca
+        self.returnUrl = returnUrl
+        self.cancelUrl = cancelUrl
+    }
+}
+
+internal struct Attributes: Codable {
```

Comment:
Same with these. We want newcomers to this project to instinctively know how to find the definition for this `Attributes` struct. If we put it in an `Attributes.swift` file it'll be easy to find.

On the other hand there may be times where a struct only applies to a specific response type, in that case it makes sense to nest it within a parent struct.